client/src/pages/Location.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components/macro";
+import PageHeadline from "../components/Header/PageHeadline";
+import platzhalter from "../assets/platzhalter.jpeg";
```

Comment:
looks kinda weird, since everything in your code that is not visible for Users is in english and suddenly there is a variable `platzhalter`. I would suggest renaming it to `placeholderImg` or something like that, so its easier to understand, what it's for.

Reply:
OK!

Reply:
every `platzhalter` is now a  `placeholder` 51b5300